workshops/personal_website/README.md

```diff
@@ -26,53 +26,57 @@ We'll be setting up GitHub and Cloud9 in this section.
 
 GitHub is a website used by millions of programmers to collaborate on code. We'll be using it to store and manage our code in Hack Club.
 
-1. Open https://github.com/join in a new tab and create a new account
-  - Choose a username you won't regret. Your actual name is a solid bet.
-  - Use an email that you have access to; you'll need to access it to verify your account
-  - Make sure to remember your username and password; you'll need them at every club meeting
-  - When prompted to choose a plan, choose the "Free" plan
-2. Verify your email by checking the inbox of the email you used
-3. Open https://gh.hackclub.com in a new tab and star the project by clicking the button that looks like this on the top right: ![](img/github_star.png)
+Go to https://github.com/join and enter your:
+
+- username (if you can't decide on one, your first and last name are a solid choice)
+- password (you'll need to remember this!)
+- email (you'll be verifying your email in the next step)
+
+![](img/gh_join.gif)
 
 _Pro tip: if you want a username that's already taken, you can often get it by [contacting GitHub](https://github.com/contact) (this is actually how we got `hackclub` on GitHub). Make sure that the username isn't actively used by another user before reaching out. [Click here](https://help.github.com/articles/name-squatting-policy/) for more details._
 
+Dash through the rest of the signup process:
+
+![](img/gh_signup_1.gif)
+![](img/gh_signup_2.gif)
+
+After you've done that, verify your email!
+
 ### 2) Creating Your First GitHub Repository
 
-GitHub allows us to host our website using a service called GitHub Pages. This means that we can put the files of our website on GitHub and GitHub will give us a URL that we can share with the world.  
+GitHub allows us to host our website using a service called GitHub Pages. This means that we can put the files of our website on GitHub in something known as a "repository," and GitHub will give us a URL that we can share with the world.
```

Comment:
Someone was like "I don't even know what this is" when I said the word repository.